docs/how-to-build.md

```diff
@@ -133,3 +133,18 @@ server distribution package.
     ```
 
    This creates `gravitino-trino-connector-{version}.tar.gz` and `gravitino-trino-connector-{version}.tar.gz.sha256` under the `distribution` directory.
+
+6. Assemble the Gravitino Spark connector package
+
+   ```shell
+    ./gradlew assembleSparkConnector
+    ```
+
+   or
+
+    ```shell
+    ./gradlew assembleDistribution
+    ```
```

Comment:
Why do we need another command to create a Spark runtime jar? I think a simple `build` should be enough.

Reply:
ok